refract-spec.md

```diff
@@ -534,7 +518,7 @@ Additionally, the `select` element is provided to give multiple possibilities fo
 
 The `extend` element provides a way to a way to multiply inherit one or more elements to form a new element. The `extend` element MUST NOT affect the original elements being extended, and MUST define a new instance of an element.
 
-The `extend` element MUST do a deep merge of the elements found in the `content`, but MUST NOT include the `id`, `namespaces` and `prefix` meta attributes from the content elements in the final element. Each element found in the `content` MUST derive from the same primitive element. The elements are merged from first to last.
+The `extend` element MUST do a deep merge of the elements found in the `content`, but MUST NOT include the `id` meta attribute from the content elements in the final element. Each element found in the `content` MUST derive from the same primitive element. The elements are merged from first to last.
```

Comment:
I think the main question here is, What happens when 2 elements from different profiles are named the same? How do I differentiate between them? Answer to that question should give the answer to the above question.